doc/extension_store/README.md

```diff
@@ -0,0 +1,76 @@
+# Proposal for Extension Store Manifest
```

Comment:
General question, why is this in the postgresql-virtual-schema repo?

Reply:
Because @redcatbear proposed to put it here.